tool/tctl/common/resource_command.go

```diff
@@ -644,6 +642,66 @@ func (rc *ResourceCommand) createNode(ctx context.Context, client auth.ClientI,
 	return trace.Wrap(err)
 }
 
+func (rc *ResourceCommand) createOIDCConnector(ctx context.Context, client auth.ClientI, raw services.UnknownResource) error {
```

Comment:
Fetching and deleting OIDC/SAML connectors is already in open-source tctl, right?

Reply:
Yeah, it kind of always "just worked" because we only explicitly check the kind on create calls.